### ISSUE TITLE
Req: Change Datetime Picker for client view in open ticket for custom form

### ISSUE THREAD

**Issue by johnkalangha** [luffynando/HawkOsTicket#8]
![image](https://user-images.githubusercontent.com/30627667/52553117-857cf900-2e1d-11e9-8841-f5fa776c4056.png)
like this datetime picker

**Comment by luffynando** [comment]
idk, only in client side modify core, but i dont understand, do you need in your form custom?

**Comment by johnkalangha** [comment]
yes sir, cause it easier to find the date if it is in that design, the previous/default, it so hard to find exact date unless you need to type, but whats the purpose of putting datetime range while you just need to type in form?

**Comment by luffynando** [comment]
Client side or admin side?

**Comment by johnkalangha** [comment]
client side sir

**Comment by johnkalangha** [comment]
any update sir? thank you
